lib/widgets/board_icon.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/widgets.dart';
+
+import 'cell_icon.dart';
+import '../model/game.dart';
+
+class BoardIconView extends StatelessWidget {
```

Comment:
Can you add doc comment to explain what this widget is showing?
Actually, we need this for all our classes, to make code reading easier for new people.